### ISSUE TITLE
Closing tab by pressing 'd' should return to the previous tab

### ISSUE THREAD

**Issue by sholland1** [vimperator/vimperator-labs#435]
To reproduce:
1. Open a new instance of Firefox
2. Press Ctrl-T to open a new tab (there should now be two tabs)
3. Switch back to the first tab
4. Press Ctrl-T to open a third tab (you should be on the third tab)

From here closing the tab by clicking the (X) button on the tab returns you to the first tab, but pressing 'd' returns you to the tab to the left of the closed tab.

Pressing 'd' should do the same thing as clicking the (X) button.

**Comment by timss** [comment]
Until a fix is available, a workaround is to map `d` to `<C-w>` like so: `noremap d <C-w>`.

**Comment by gkatsev** [comment]
I'm not sure that we would fix it. `d` is defined as close the tab and go left. `D` is close the tab and go right.
We possibly would want an option to do the same behavior as `<c-w>` but the default shouldn't change as there are probably people relying on that behavior as well.

**Comment by timss** [comment]
Good point, didn't think of that. It would also merit changes to `tabclose` etc. if were to be changed.

There's also `tabclose -select lastactive`, which mimics the behavior of `<C-w>`.

Maybe a simple user map is the best solution then.

**Comment by gkatsev** [comment]
Yeah, I'm thinking maybe an option is necessary because this gets asked a lot.

**Comment by timss** [comment]
`wishlist++` :-)

**Comment by sholland1** [comment]
Thanks for the workaround. It works great! :smile:

**Comment by Tarnasa** [comment]
Use `:q` instead of `d` to return to the previously accessed tab.
As was stated above, `d` is specifically defined as: "Close and move right" as can be seen in: liberator://help/tabs
